osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/openstack/securitygroup/CheckK8sSecurityGroupLabelMetaTask.java

```diff
@@ -18,39 +18,65 @@
 
 import javax.persistence.EntityManager;
 
-import org.apache.commons.lang.NotImplementedException;
-import org.osc.core.broker.job.Task;
+import org.apache.log4j.Logger;
```

Comment:
Can you move all the k8s specific tasks  to  org.osc.core.broker.service.tasks.conformance.k8s.securitygroup; (like I did for deployment spec)? Previously when you ask I meant that you can keep here in the openstack package all the existing ones (even the ones you update for k8s) we can improve those later. But new ones k8s specific can be in the right package.